src/screens/SiteDetailScreen.js

```diff
@@ -278,29 +299,31 @@ class SiteDetailScreen extends Component {
                         </TouchableOpacity>
 
                         <View style={contentContainerStyle}>
-                            <Text style={[textStyle, bottomMargin]}>
+                            <Text style={[textStyle, bottomMargin, topMargin]}>
                                 {description}
                             </Text>
 
                             <Text style={sectionTitleStyle}>
-                                {campsite_form.accessibility}
+                                {campsite_form.price}
                             </Text>
                             <Text style={[textStyle, bottomMargin]}>
-                                {campsite_form.accessibility_options[accessibility]}
+                                {campsite_form.price_options[price]}
                             </Text>
 
                             <Text style={sectionTitleStyle}>
-                                {campsite_form.nearest_town}
+                                {campsite_form.accessibility}
                             </Text>
                             <Text style={[textStyle, bottomMargin]}>
-                                {nearestTown}
+                                {campsite_form.accessibility_options[accessibility]}
                             </Text>
 
+                            {this.renderForestInfo()}
+
                             <Text style={sectionTitleStyle}>
-                                {campsite_form.price}
+                                {campsite_form.nearest_town}
                             </Text>
                             <Text style={[textStyle, bottomMargin]}>
-                                {campsite_form.price_options[price]}
+                                {nearestTown}
```

Comment:
GH-89 -- reorganized the flow of items on `SiteDetailPage`. I think it finally makes more sense now.